packages/core-database/src/repositories/utils/search-parameter-converter.ts

```diff
@@ -42,14 +41,15 @@ export class SearchParameterConverter implements Database.IISearchParameterConve
     }
 
     private parseOrderBy(searchParameters: Database.ISearchParameters, orderBy?: any) {
-        if (orderBy && typeof orderBy === "string") {
-            const fieldDirection = orderBy.split(":").map(o => o.toLowerCase());
-            if (fieldDirection.length === 2 && (fieldDirection[1] === "asc" || fieldDirection[1] === "desc")) {
-                searchParameters.orderBy.push({
-                    field: snakeCase(fieldDirection[0]),
```

Comment:
This actually might have to stay :thinking: but currently it has no effect since the value is lowercased above